### PR TITLE
Songbook implementation

### PR DESCRIPTION
**Pull request by tonitajger**
Songbook implementation with the following nice features:
- _Contact list_ where the songs are sorted in groups by the first letter of the song title
- Easy to use _search bar_ for greater user experience
- Very nice looking _preloaders_ for farmers and peasants with bad internet connection

Hope you like the songbook! I'm very happy with this work and want to credit my fellow supervisor _Fredrik Lastow_ aka Frallan. Hope you have a good weekend, now I will go to Fredmans! :)

### CODE REVIEW

File: www/js/songbook.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    console.log(resp.statusText);
+  });
+});
+
+//Creates a list of JSON objects containing list and the first letter of the songs
+function initSongList(songdata) {
+  //Remove sort when new song API is live
+  songdata.sort(function(a, b) {
+    return a.title.localeCompare(b.title);
+  });
```
Comment (FredrikLastow) [inline www/js/songbook.js]: The API already sorts the songs xD

**Comment by olofenglund96** [comment]
Precious songs... 

+++ <><   <><   <><  a small loan of 1 million fishes

**Comment by Deladox** [comment]
Perfekt addition till fappen, nästan lika bra som mina egna, fortsätt jobba så blir du lika bra som jag snart Tony!

File: www/index.html [hunk 2]
```diff
@@ -198,6 +200,20 @@
         <div class="pages navbar-fixed">
           <div data-page="tab5" class="page">
             <div class="page-content">
+            <!--<div class="content-block-title">Din mamma</div>-->
```
Comment (FredrikLastow) [inline www/index.html]: Remove this too lol

**Pull request merged**